src/test/ui/suggestions/return-without-lifetime.stderr

```diff
@@ -0,0 +1,19 @@
+error[E0106]: missing lifetime specifier
+  --> $DIR/return-without-lifetime.rs:3:34
+   |
+LL | fn func1<'a>(_arg: &'a Thing) -> &() { unimplemented!() }
+   |                                  ^ help: consider using the named lifetime `'a`: `&'a`
```

Comment:
```suggestion
   |                                  ^ help: consider using the named lifetime: `&'a`
```
I think it would make sense here not to explicitly mention the named lifetime, because it's obvious from the reference that immediately follows.